spring-rabbit/src/main/java/org/springframework/amqp/rabbit/core/RabbitTemplate.java

```diff
@@ -1566,7 +1627,8 @@ public void addListener(Channel channel) {
 			if (this.publisherConfirmChannels.putIfAbsent(key, this) == null) {
 				publisherCallbackChannel.addListener(this);
 				if (logger.isDebugEnabled()) {
-					logger.debug("Added pubsub channel: " + channel + " to map, size now " + this.publisherConfirmChannels.size());
+					logger.debug("Added confirmable channel: " + channel + " to map, size now "
```

Comment:
I don't know this `confirmable` word :smile: 

Maybe `publisher-confirm` would be better? 
At least that is how `PublisherCallbackChannelImpl.toString()` represents it...